components/blog/ui/ArticleContainer.tsx

```diff
@@ -29,16 +30,27 @@ const ArticleContainer = (props: ArticleContainerProps) => {
   const { team } = useParams();
   const { articleListData, thumbnail, description } = props;
   const FilteredCategoryList = useGetCategoryList(team);
+  const CategorySelected = useGetCategory();
 
   if (!FilteredCategoryList) return <LoadingLottie width={10} height={10} fit />;
 
   const LiteralList = getLiteralCategoryList(FilteredCategoryList);
 
-  //아티클 리스트가 없고 블로그 대문이 있을 때
+  //카테고리 선택됨, 아티클 리스트 없고 블로그 대문 있을 때
+  if (CategorySelected !== 'home' && articleListData.length === 0 && thumbnail)
+    return (
+      <BlogImgContainer>
+        <BlogImg thumbnail={thumbnail} description={description} />
+        <CategoryBtnWrapper>
+          <CategoryBtnBar LiteralList={LiteralList} />
+        </CategoryBtnWrapper>
+      </BlogImgContainer>
+    );
+
+  //아티클 리스트가 없고 카테고리 선택 안되고 블로그 대문이 있을 때
```

Comment:
큰 차이는 없지만!
```suggestion
  if (articleListData.length === 0 && thumbnail) {
    if(CategorySelected !== 'home') {
       return (
          <BlogImgContainer>
            <BlogImg thumbnail={thumbnail} description={description} />
            <CategoryBtnWrapper>
              <CategoryBtnBar LiteralList={LiteralList} />
            </CategoryBtnWrapper>
          </BlogImgContainer>
        );
    } else {
      return (
      <BlogImgContainer>
        <BlogImg thumbnail={thumbnail} description={description} />
      </BlogImgContainer>
     }
    }
```

아래도 그렇고 뭔가 이렇게 조건 분기가 많은 경우에는 겹치는 조건 로직을 하나로 다뤄줄 수 있으면 더 깔끔하고 유지보수 하기에도 더 좋을 것 같다는 의견입니다 :) 코딩 스타일 공유!

+ 그리고 if 안에서 걸리면 return 해줘버리기 때문에 else까지는 안 써도 괜찮겠네용 취향차이 ~.~

Reply:
아ㅏㅏ 안그래도 이거 뭔가 합칠 수 있을 거 같은데~~ 햇는데 감사합니둥~